registry/config.yaml

```
@@ -0,0 +1,36 @@
+
+# Logging settings
+logging:
+  level: "${LOG_LEVEL:-info}"   # Options: debug, info, warn, error
+  log_file: "${LOG_FILE:-build.log}"  # Log file for build process
+
+# Optional: Proxy settings for Docker registry (if behind a proxy)
+proxy:
+  enabled: ${PROXY_ENABLED:-false}  # Set to true if behind a proxy
+  url: "${PROXY_URL:-http://proxy.example.com:8080}"  # Proxy URL
+  username: "${PROXY_USERNAME}"  # Proxy username if required
+  password: "${PROXY_PASSWORD}"  # Proxy password if required
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)